templates/README.md

```diff
@@ -0,0 +1,28 @@
+# Using templates to implement Secretless Connector Plugins
+
+We created connector templates to ease the process of adding new connectors to secretless.
+Before using the templates to add new connector plugins, be sure to read the [Secretless Connector Plugins README](https://github.com/cyberark/secretless-broker/blob/master/pkg/secretless/plugin/connector/README.md)
+
+To add a new connector do the following:
+
+1. Copy the relevant template directory (HTTP/TCP) into `internal/plugin/connectors/<protocol>`. 
+    1. Inside each template directory you will find the required files & structs implemented, with
+instructions to fill them with the content of the new connector
```

Comment:
are the instructions in the form of TODOs or otherwise? would be good to be clear about how to find the instructions here